sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -0,0 +1,98 @@
+        public virtual Azure.Pageable<Azure.Security.KeyVault.Administration.Models.RoleDefinition> GetRoleDefinitions(string scope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.Security.KeyVault.Administration.Models.RoleDefinition> GetRoleDefinitions(System.Uri scope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.Security.KeyVault.Administration.Models.RoleDefinition> GetRoleDefinitionsAsync(string scope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.Security.KeyVault.Administration.Models.RoleDefinition> GetRoleDefinitionsAsync(System.Uri scope, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+    }
+    public partial class AccessControlClientOptions : Azure.Core.ClientOptions
+    {
+        public AccessControlClientOptions(Azure.Security.KeyVault.Administration.AccessControlClientOptions.ServiceVersion version = Azure.Security.KeyVault.Administration.AccessControlClientOptions.ServiceVersion.V7_2_Preview) { }
+        public Azure.Security.KeyVault.Administration.AccessControlClientOptions.ServiceVersion Version { get { throw null; } }
+        public enum ServiceVersion
+        {
+            V7_2_Preview = 1,
+        }
+    }
+    public static partial class KeyVaultAdministrationConstants
+    {
+    }
+}
+namespace Azure.Security.KeyVault.Administration.Models
+{
+    public partial class KeyVaultPermission
+    {
+        internal KeyVaultPermission() { }
+        public System.Collections.Generic.IReadOnlyList<string> Actions { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<string> DataActions { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<string> NotActions { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<string> NotDataActions { get { throw null; } }
+    }
+    public partial class RoleAssignment
+    {
+        internal RoleAssignment() { }
+        public string Id { get { throw null; } }
+        public string Name { get { throw null; } }
+        public Azure.Security.KeyVault.Administration.Models.RoleAssignmentPropertiesWithScope Properties { get { throw null; } }
+        public string Type { get { throw null; } }
+    }
+    public partial class RoleAssignmentListResult
+    {
+        internal RoleAssignmentListResult() { }
+        public string NextLink { get { throw null; } }
+        public System.Collections.Generic.IReadOnlyList<Azure.Security.KeyVault.Administration.Models.RoleAssignment> Value { get { throw null; } }
+    }
+    public partial class RoleAssignmentProperties
```

Comment:
Can we combine these and make them easier to construct? If `Scope` is null, then there's no scope. For construction, can they pass in a `RoleDefinition` instead? How do they get the `principalId`? As a first attempt this is fine, but these are things about which we should have a broader discussion that could impact management-plane.

Reply:
tracking this in #12613